db/migrate/20180413193115_require_title_person.rb

```diff
@@ -0,0 +1,6 @@
+class RequireTitlePerson < ActiveRecord::Migration
+  def change
+    change_column_null :people, :title, false, "Recruit"
```

Comment:
I worry some about putting this into the database. I would like to define the titles in the database so that other groups can define their own. They may not have any title of `recruit`. Perhaps if we changed this to `Unknown`, or something along those lines.

Reply:
@kgf As of right now, all logic determining events, people and there availabilities requires having a rank of 'Recruit' or higher or is will not calculate them as an availability. I can make an unknown rank as default but the default title_order number will have to be equal to 'Recruit' for everything to work properly.

Reply:
Yes